code/modules/paperwork/pen.dm

```diff
@@ -195,15 +189,15 @@
  * Sleepypens
  */
 
-/obj/item/pen/sleepy/attack(mob/living/M, mob/user)
-	if(!istype(M))
+/obj/item/pen/sleepy/attack(mob/living/M, mob/user, params)
+	. = ..()
+	if(!.) //If we didn't stab them, no injection.
 		return
-
-	if(..())
-		if(reagents.total_volume)
-			if(M.reagents)
-
-				reagents.trans_to(M, reagents.total_volume, transfered_by = user, methods = INJECT)
+	if(!reagents.total_volume) //If we don't have reagents in the pen, no injection.
+		return
+	if(!M.reagents) //If the target doesn't have a reagents container, no injection.
+		return
```

Comment:
These comments are useless--they just repeat what the code is already saying. Comments should say why, not what. Just remove them.